test/screenshot/backstop.js

```diff
@@ -0,0 +1,72 @@
+const { readdirSync } = require('fs')
+const { join, parse } = require('path')
+
+const BASE_URL = 'http://127.0.0.1:9001/iframe.html'
+
+function deKebab(text) {
+  return text
+    .replace(/([A-Z])/g, ' $1')
+    .split(' ')
+    .map((word = '') => {
+      const first = (word[0] || '').toUpperCase()
+
+      return `${first}${word.slice(1)}`
+    })
+    .join(' ')
+}
+
+// TODO: refactor to use existing getName function from storyFolders
+function getName(path) {
+  const fileName = path.replace(/^.*[\\/]/, '').replace(/\.js$/, '')
+
+  return deKebab(fileName)
+}
+
+const stories = readdirSync(join(__dirname, '../../stories'))
+  // TODO: refactor file structure layout so that story definitions are isolated to own directory
+  // (ie, move core, static, and index.js elsewhere)
+  .filter(path => ['grid', 'layout', 'components'].indexOf(path) >= 0)
```

Comment:
good point, we may want to move them to `/storybook`. This will require small changes on the `gh-pages` deploy script. Mentioning it just so that we don't forget